main.tf

```diff
@@ -18,9 +18,10 @@ terraform {
 resource "myprovider_address_group" "consul_service" {
   for_each = local.consul_services
 
-  name     = "${var.address_group_prefix}${each.key}"
-  tags     = var.address_group_tags
-  policies = [each.value.cts_user_defined_meta["policy_name"]]
+  name         = "${var.address_group_prefix}${each.key}"
+  description  = format("Address group for: %s", join(", ", keys(var.catalog_services)))
```

Comment:
I wanted to update this example to make use of the `var.catalog_services`. But it's a bit awkward since the keys of the map are the same as `local.consul_services` (which is a good example of local variables). Happy to remove it if others find it awkward or have suggestions on how to better demonstrate usage of `var.catalog_services`